Source/Request.swift

```diff
@@ -80,6 +80,8 @@ open class Request {
         var uploadProgressHandler: (handler: ProgressHandler, queue: DispatchQueue)?
         /// `ProgressHandler` and `DispatchQueue` provided for download progress callbacks.
         var downloadProgressHandler: (handler: ProgressHandler, queue: DispatchQueue)?
+        /// `RetryHandler` provided for redirect responses.
+        var redirectHandler: RedirectHandler?
```

Comment:
Per our previous conversations, do we want this to be a single optional instance or an array, to allow for easier compensation of multiple handlers?

Reply:
After discussion in Slack, we've decide that these should stay a single instance and use a `precondition` in the chained API to prevent clients from calling it twice.